Tools/CISupport/Shared/maybe-delete-directory.bash

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Deletes the directory given as the first argument if and only if it is greater than 10GB in size
+
+if [[ $# < 1 ]]
+then
+    echo "Must provide absolute path to directory to prune."
+    exit 1
+fi
+
+if [[ -L "${1%/}" ]]
+then
+    echo "$1 must not be a symbolic link."
```

Comment:
Similarly, I'd have the script say that it `$1` is a symlink, please provide a direct directory link (not quite sure why symlinks aren't OK though).